README.md

```diff
@@ -108,9 +120,14 @@ See [this](https://github.com/flutter/flutter/issues/46481) for more info on the
 
 See also:
 
-- [Material UI for Flutter](https://flutter.dev/docs/development/ui/widgets/material)
-- [Cupertino UI for Flutter](https://flutter.dev/docs/development/ui/widgets/cupertino)
-- [MacOS UI for Flutter](https://github.com/GroovinChip/macos_ui)
+- [Material UI for Flutter](https://flutter.dev/docs/development/ui/widgets/material) (built-in)
+- [Cupertino UI for Flutter](https://flutter.dev/docs/development/ui/widgets/cupertino) (built-in)
+- [MacOS UI for Flutter](https://github.com/GroovinChip/macos_ui) ([pub](https://pub.dev/packages/macos_ui)) ([demo](https://groovinchip.github.io/macos_ui))
+- [Stockholm (MacOS-like) UI for Flutter](https://github.com/serverpod/stockholm) ([pub](https://pub.dev/packages/stockholm)) ([demo](https://serverpod.dev/stockholm))
```

Comment:
nah

Reply:
do I remove the WIP ones only or the pub and demo links too ?

Reply:
yup! leave only material and cupertino

Reply:
Done in [17bc4c3](https://github.com/bdlukaa/fluent_ui/pull/400/commits/17bc4c3fd7476784850387be07bf1989ceeeefa9)